hip/CMakeLists.txt

```diff
@@ -1,20 +1,29 @@
-if(NOT DEFINED HIPBLAS_PATH)
-    if(NOT DEFINED ENV{HIPBLAS_PATH})
-        set(HIPBLAS_PATH "${HIP_PATH}/../hipblas" CACHE PATH "Path to which HIPBLAS has been installed")
+if(NOT DEFINED ROCM_PATH)
+    if(DEFINED ENV{ROCM_PATH})
+        set(ROCM_PATH $ENV{ROCM_PATH} CACHE PATH "Path to which ROCM has been installed")
+    elseif(DEFINED ENV{HIP_PATH})
+        set(ROCM_PATH "$ENV{HIP_PATH}/.." CACHE PATH "Path to which ROCM has been installed")
     else()
+        set(ROCM_PATH "/opt/rocm" CACHE PATH "Path to which ROCM has been installed")
+    endif()
+endif()
+
+if(NOT DEFINED HIPBLAS_PATH)
+    if(DEFINED ENV{HIPBLAS_PATH})
```

Comment:
I forgot to mention it before. In hipblas/hipsparse, they use `if (DEFINED $ENV{})` https://github.com/tcojean/hipBLAS/blob/ad14b3bf5ec6da92abd646a533268aace33e0902/CMakeLists.txt#L9
which should be `if (DEFINED ENV{})` here

Reply:
Good find. Maybe they fixed it upstream. I need to update these repositories.